### PR TITLE
Appealsv2 appeal list item v2 tests

### PR DESCRIPTION
**Pull request by markgreenburg**
Adds unit tests for AppealListCardV2 component.

Also, snuck in a change to pass the veteran's full name into `getStatusContents ` in the same component, as this arg is needed to construct the user's name in the `death` current status type's text snippet.

### CODE REVIEW

**Review by cvalarida — APPROVED** [review]
What @markgreenburg had before I added more tests looked good. Can you take a look, @kreek since I added to this?

**Comment by markgreenburg** [comment]
@cvalarida @kreek I'll review :)

**Comment by cvalarida** [comment]
~He's a ghost! Don't believe him!!~
Thanks kindly!

File: test/claims-status/components/appeals-v2/AppealListItemV2.unit.spec.jsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+    expect(issuesText).to.not.contain('Issues');
+  });
+
+  it('should say "issues" if there are multiple issues on appeal', () => {
+    const wrapper = shallow(<AppealListItemV2 {...defaultProps}/>);
+    const issuesText = wrapper.find('.card-status + span').first().text();
+    expect(issuesText).to.contain('Issues');
+  });
+
+  it('should create a link to the appeal status page', () => {
```
Comment (markgreenburg) [inline test/claims-status/components/appeals-v2/AppealListItemV2.unit.spec.jsx]: well that seems like a 'duh' test in retrospect. Thanks for adding.

**Comment by kreek** [comment]
@markgreenburg 👶 not allowed 🍼

File: test/claims-status/components/appeals-v2/AppealListItemV2.unit.spec.jsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    const wrapper = shallow(<AppealListItemV2 {...defaultProps}/>);
+    expect(wrapper.find('Link').first().props().to)
+      .to.equal(`appeals-v2/${defaultProps.appeal.id}/status`);
+  });
+
+  it('should not show the issue text if no description is given', () => {
```
Comment (markgreenburg) [inline test/claims-status/components/appeals-v2/AppealListItemV2.unit.spec.jsx]: Yep, makes total sense.

**Review by markgreenburg — COMMENTED** [review]
LGTM, thanks for finishing this up!

**Review by kreek — APPROVED** [review]
LGTM 🚀

**Pull request merged**